iron-fit-behavior.html

```diff
@@ -485,14 +485,19 @@
     },
 
     __getNormalizedRect: function(target) {
-      if (target === document.documentElement || target === window) {
+      if (target === window) {
```

Comment:
Looks like everything else is scaled accordingly (`getBoundingClientRect()` does the magic), but not `innerWidth/innerHeight`, and that's the actual viewport. So we have to compute the zoom factor and what the scrolling on the viewport + use the scrolling of the document.... 😭 

Here a simplified version of #61: a button with a top margin  http://jsbin.com/sajibu/7/edit?html,output

With the version of master the dropdown is shrunk in height
![zooming-issue](https://cloud.githubusercontent.com/assets/6173664/18022990/e20122da-6ba8-11e6-88d0-285a7f5cd053.gif)

With this fix it is not:
![zooming-issue-fix](https://cloud.githubusercontent.com/assets/6173664/18022993/e8a6d2c4-6ba8-11e6-9533-73ea728e65cc.gif)